### PR TITLE
chore(*): make this repo a thin wrapper of Kong docker images

### PR DESCRIPTION
**Pull request by fffonion**
KAG-532

### CODE REVIEW

File: Dockerfile.apk [hunk 1]
```diff
@@ -1,46 +1,17 @@
 # When you update this file substantially, please update build_your_own_images.md as well.
-FROM alpine:3.17
+FROM kong/kong:3.3.0-alpine@sha256:56499004203a618ab19aceca85e6611d3baba731b9577da27dee6bb6c8187cf5
 
 LABEL maintainer="Kong Docker Maintainers <docker@konghq.com> (@team-gateway-bot)"
 
 ARG KONG_VERSION=3.3.0
 ENV KONG_VERSION $KONG_VERSION
 
-ARG KONG_AMD64_SHA="494522d5ef9baf674272bbb7075e406a4515a7475253fd3026cc7ca9451612a2"
```
Comment (kikito) [inline Dockerfile.apk:9]: These changes will break the (already quite broken, admittedly) the process for updating the official images to Docker Images (update.sh and submit.sh).

In particular, those scripts parse these Dockerfiles grepping for KONG_AMD64_SHA currently.

I recommend either changing those files to accomodate these changes or coming up with a different way to submit to the official docker images if that's not feasible.

**Review by kikito — CHANGES_REQUESTED** [review]

File: Dockerfile.apk (hunk 1 above)
Comment (fffonion) [reply Dockerfile.apk:9]: @kikito good point, i will update this PR to change the release script as well

Comment (fffonion) [reply Dockerfile.apk:9]: fixed

**Comment by kikito** [comment]
Reinstated in #664 in draft mode, for future improvements/research

**Pull request merged**